frontend/src/lib/components/leaderboard.svelte

```diff
@@ -26,10 +27,13 @@
         </Table.Row>
       </Table.Header>
       <Table.Body>
-        {#each data as { loses, name, wins, mmr, winningStreak, losingStreak }, idx}
+        {#each data as { userId, loses, name, wins, mmr, winningStreak, losingStreak }, idx}
           <Table.Row>
             <Table.Cell class="w-16 font-bold">{idx + 1}</Table.Cell>
-            <Table.Cell>{name}</Table.Cell>
+            <Table.Cell
+              title={users?.find((user) => user.userId == userId)
+                ?.displayName ?? undefined}>{name}</Table.Cell
+            >
```

Comment:
Can we show the full displayName instead of name? Maybe `displayName (name)`, and to prevent it taking up too much space then set a max width and text ellipsis overflow?

Reply:
<img width="1002" alt="image" src="https://github.com/madsthom/mmr-project/assets/2279254/08142677-117f-4026-923e-4f0c3ebd28d9">
We could do something like this instead. I am hiding full name on mobile for now. The user can always switch to landscape for the full table.